Description: Don't compile the Maven plugin (already provided by jarjar-maven-plugin)
Author: Emmanuel Bourg <ebourg@apache.org>
Forwarded: not-needed
--- a/build.xml
+++ b/build.xml
@@ -72,6 +72,7 @@
               debug="${compile.debug}" optimize="${compile.optimize}" deprecation="${compile.deprecation}">
                  <classpath refid="@{refid}"/>
                  <compilerarg value="-Xlint"/>
+                 <exclude name="**/*Mojo*"/>
             </javac>
         </sequential>
     </macrodef>
